docs/generate/readme.md

```diff
@@ -0,0 +1,397 @@
+# <img align="center" src="../images/logo.png">  Generating Clients with AutoRest
+
+This guide tells you how to generate code from your OpenAPI definition using AutoRest. We'll take this incrementally, working
+on first how to generate a single file, then how to generate with a configuration file, and keep taking it from there.
+
+The command line usage of AutoRest boils down to the following:
+> `autorest [config-file.md | config-file.json | config-file.yaml] [additional options]`
+We'll be building upon this in our individual sections.
```

Comment:
I think this needs an additional newline because it's getting included with the 'quote' block in the rendered page

```suggestion

We'll be building upon this in our individual sections.
```